### DISCUSSION TITLE
Unable to open sample project on android/IOS

### DISCUSSION THREAD

**Discussion by Marc-Antoine-Soucy** [unoplatform/uno.extensions#972] [Q&A] [answered]
After running this command 
 dotnet new -i Uno.Extensions.Templates
and this command
dotnet new unoapp-extensions -o SampleName, when I try to open the app with IOS or Android

I get this when I set the head to mobile
![image](https://user-images.githubusercontent.com/90481654/206037951-2c0a44f0-c1e9-4a20-9d40-a13baa6697e5.png)
 instead of this
 
![image](https://user-images.githubusercontent.com/90481654/206039360-e60fecff-8fd7-4d60-b810-d4d12a499ab1.png)

I am using VS 2022 Version 17.4.1,
I ran Uno-check 1.10.0,
I ran git clean -fdx, 
I tried restarting my computer, 
and I know it's not that VS cannot detect my phone, since with the default uno .net-6 app, it finds my phone just fine. 

Is there something else I could do to debug this?

**Comment by dansiegel** [comment]
This is a result of the Central Package Management. If you grab a newer version of the template it is disabled by default.

**Reply by Marc-Antoine-Soucy** [reply]
Thanks for the answer, but, I already tried to run the command dotnet new -i Uno.Extensions.Templates (which told me it updated the sample to the latest one) before making a new sample and I still had the same issue. Is there another command I'm supposed to use to update the template?

**Comment by Soap-141** [comment]
I'm having the same issue, I tried removing every targets except for iOS and I was able to see the list of device on the mobile head. But only unsupported devices, then I tried added the `UseMaui` option and set it to true and I was seeing all my devices but I couldn't build and deploy.

**Reply by jeromelaban** [reply]
Which version of VS are you using?

**Comment by Marc-Antoine-Soucy** [comment, marked as answer]
Well, seems like all I had to do was update the uno.winui nugget. but, it should probably be changed so that the uno.extension template works out of the box?

**Comment by nickrandolph** [comment]
New version of templates are available that should work out of the box!!!
